src/ErlPL-1.1.xml

```diff
@@ -0,0 +1,325 @@
+<spdx name="Erlang Public License v1.1" identifier="ErlPL-1.1" osi-approved="false">
+  <urls>
+    <url>http://www.erlang.org/EPLICENSE</url>
+  </urls>
+  <notes>This Erlang License is a derivative work of the Mozilla Public License, Version 1.0. It contains terms which
+         differ from the Mozilla Public License, Version 1.0.</notes>
+  <license>
+    <title>
+      <p>ERLANG PUBLIC LICENSE Version 1.1</p>
+    </title>
+    <body>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>Definitions.</p>
+        </li>
+        <li>
+          <b>1.1.</b>
+          <p>``Contributor&apos;&apos; means each entity that creates or contributes to the creation of
+             Modifications.</p>
+        </li>
+        <li>
+          <b>1.2.</b>
+          <p>``Contributor Version&apos;&apos; means the combination of the Original Code, prior Modifications
+             used by a Contributor, and the Modifications made by that particular Contributor.</p>
+        </li>
+        <li>
+          <b>1.3.</b>
+          <p>``Covered Code&apos;&apos; means the Original Code or Modifications or the combination of the
+             Original Code and Modifications, in each case including portions thereof.</p>
+        </li>
+        <li>
+          <b>1.4.</b>
+          <p>``Electronic Distribution Mechanism&apos;&apos; means a mechanism generally accepted in the
+             software development community for the electronic transfer of data.</p>
+        </li>
+        <li>
+          <b>1.5.</b>
+          <p>``Executable&apos;&apos; means Covered Code in any form other than Source Code.</p>
+        </li>
+        <li>
+          <b>1.6.</b>
+          <p>``Initial Developer&apos;&apos; means the individual or entity identified as the Initial
+             Developer in the Source Code notice required by Exhibit A.</p>
+        </li>
+        <li>
+          <b>1.7.</b>
+          <p>``Larger Work&apos;&apos; means a work which combines Covered Code or portions thereof with code
+             not governed by the terms of this License.</p>
+        </li>
+        <li>
+          <b>1.8.</b>
+          <p>``License&apos;&apos; means this document.</p>
+        </li>
+        <li>
+          <b>1.9.</b>
+          <p>``Modifications&apos;&apos; means any addition to or deletion from the substance or structure of
+             either the Original Code or any previous Modifications. When Covered Code is released as a
+             series of files, a Modification is:</p>
+        </li>
+        <list>
```

Comment:
This list needs to be made a child of the list above. Similarly for other lists throughout this file. Pull request submitted.